config/etc/system-image/writable-paths

```diff
@@ -19,7 +19,8 @@
 /var/snap                              auto                    persistent  transition  none
 /var/lib/bluetooth                      auto                    persistent  none        none
 # cloud-init
-/etc/cloud                              auto                    persistent  none        none
+/etc/cloud                              auto                    synced  none        none
+/etc/cloud/cloud.cfg.d                  auto                    synced  none        none
```

Comment:
After some testing locally; it appears that's good enough.  I used only the /etc/cloud synced mode and I can populate files to /etc/cloud and /etc/cloud/cloud.cfg.d/ in the writable partition and both show up correctly.  I've updated this branch with the change.